src/ui/components/Quiz.jsx

```diff
@@ -1,20 +1,23 @@
-import "./Quiz.css"
-import { useDispatch } from "react-redux"
-import { selectedQuizAction } from "../../state/allQuizzes"
-import { Button } from "./Button"
-
-export const Quiz = ({title, id}) => {
-  const dispatch = useDispatch()
+import "./Quiz.css";
```

Comment:
is this `Quiz.css` file being used? I couldn't find it in the PR

Reply:
I forgot to delete it!